.github/workflows/build-release.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     name: mypy
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3.1.0
-      - uses: actions/setup-python@v4.3.0
+      - uses: actions/checkout@v4.0.0
+      - uses: actions/setup-python@v4.7.0
         with:
           python-version: '3.11'
```

Comment:
We might want to consider bumping the tooling to 3.12 `allow-prereleases` while we are at it.

Reply:
I don't mind either way. As long as we keep it not too old so that it runs on the CI VM.

Reply:
For tooling, how about `'3.x'`? 

It will use the "latest stable" version, currently 3.11, and will move to 3.12 in the near future. And we won't need to bump this particular line once a year.

Reply:
Sounds good, but then we lose traceability. But I see no real practical issue (it's not like using latest version of all dependencies), so I think it would be fine.